src/index.html

```diff
@@ -22,6 +22,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__item">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="/"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="/"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="icon"
+        />
+
+        <div class="item">
+          <div class="item__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="item__code">Product code: 195434</div>
```

Comment:
You need to use semantic tag for text and fix classes everywhere
```suggestion
          <h2 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h2>
          
          <p class="card__code">Product code: 195434</p>
```